docs/source/tutorials/map_styling.rst

```diff
@@ -0,0 +1,65 @@
+.. _map_styling:
+
+Cartography and Map Styling
```

Comment:
Let's call a duck a duck, and just call this section "referencing". It might even make sense to extend the scope of http://scitools.org.uk/cartopy/docs/latest/citation.html to include this useful information, rather than having it as separate documents.